Exiled.API/Features/Room.cs

```diff
@@ -76,7 +76,7 @@ public class Room : MonoBehaviour
         /// <summary>
         /// Gets a <see cref="IEnumerable{T}"/> of <see cref="Player"/> in the <see cref="Room"/>.
         /// </summary>
-        public IEnumerable<Player> Players => Player.List.Where(player => player.IsAlive && player.CurrentRoom.Transform == Transform);
+        public IEnumerable<Player> Players => Player.List.Where(player => player.IsAlive && player.CurrentRoom != null && player.CurrentRoom.Transform == Transform);
```

Comment:
``&& (!(player.CurrentRoom is null) && player.CurrentRoom.Transform == Transform)``

Reply:
hmmmm yes